app/renuo_upload_signing.rb

```diff
@@ -36,6 +32,14 @@ class RenuoUploadSigning < Sinatra::Base
 
   private
 
+  def private_key_valid
```

Comment:
i would rename it to `private_key_valid?`

Reply:
Agree.